build-system/tasks/update-packages-document-register-element.js

```diff
@@ -0,0 +1,135 @@
+/**
+ * Copyright 2019 The AMP HTML Authors. All Rights Reserved.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS-IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+const assert = require('assert');
+
+
+/**
+ * Takes a document-register-element polyfill init object (like: git.io/fhxbR)
+ * in the form:
+ *
+ *  {
+ *    'collections': {
+ *      // ...
+ *    },
+ *    'elements': {
```

Comment:
Last I looked, the entire init object wasn't actually necessary for the DRE polyfill to work, it was a left over from some previous code. Either way, this is very risky change, and it won't be necessary with CEv1 polyfill.

Reply:
1. What's the status of CEv1?
2. How exactly is it risky?